.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
+about: Create an extensive report to help us track down a problem
+
+---
+<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
+## Bug report
+**Describe the bug**
+A clear and concise description of what the bug is. A bug report that is not clear will be closed:
+<!--- Put your text below this line -->
+
+
+**To Reproduce**
+Steps to reproduce the behavior:
+<!--- Put your text below this line -->
+1. 
+2.
+3.
+
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+<!--- Put your text below this line -->
+
+**Debuglog**
+<!--- Put your text below this line -->
+The debuglog can be found here:
+
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+<!--- Put your text below this line -->
+
+
+**Platform (please complete the following information):**
+ Operating system:
+ <!--- Put your text below this line. Checkboxed can easily be ticked once issue is created -->
+ - [ ] Android
+ - [ ] iOS
+ - [ ] Linux
+ - [ ] OSX
+ - [ ] Windows
+ - [ ] Windows UWP
+
+ - Operating system version:
+ - Kodi version:
+
+
+**Additional context or screenshots (if appropriate):**
+Add any other context about the problem here.
+<!--- Put your text below this line -->
+
+
+
+
+<!--- End of this issue -->
+*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
+Team Kodi will consider your bug report however we will not make any promises if it will be fixed at some point in the future.*
```

Comment:
* "report however" -> "report, however"
* "if it will be fixed" -> "that it will be fixed"
* "at some point in the future" -> "" (not needed imo)